src/components/atoms/ExplorerLink.tsx

```diff
@@ -16,7 +16,7 @@ export default function ExplorerLink({
 }: {
   networkId: number
   path: string
-  children: ReactNode
+  children?: ReactNode
```

Comment:
Was it intended to make `children` optional? Cause we should not have any empty link which this would produce when no `children` is passed. Also there should be no further actions on the teasers as they themselves are fully clickable so no actions inside of that link can be actually used so most likely we don't want an ExplorerLink inside the account teasers

Reply:
yess, that was my bad, forgot it there